src/System.ServiceModel.Syndication/tests/System/ServiceModel/Syndication/Rss20FeedFormatterTests.cs

```diff
@@ -0,0 +1,2029 @@
+        <generator>generator</generator>
+        <a10:contributor />
+        <a10:contributor feedauthor__name1="""" d3p1:feedauthor__name2="""" d3p1:feedauthor__name3=""feedauthor__value"" d2p2:feedauthor__name4="""" xmlns:d3p1=""feedauthor__namespace"">
+            <a10:name>feedauthor__name</a10:name>
+            <a10:uri>feedauthor__uri</a10:uri>
+            <a10:email>feedauthor__email</a10:email>
+            <Rss20FeedFormatterTests.ExtensionObject xmlns:i=""http://www.w3.org/2001/XMLSchema-instance"" xmlns=""http://schemas.datacontract.org/2004/07/System.ServiceModel.Syndication.Tests"">
+                <Value>10</Value>
+            </Rss20FeedFormatterTests.ExtensionObject>
+        </a10:contributor>
+        <image>
+            <url>http://imageurl.com/</url>
+            <title>feedtitle_title</title>
+            <link />
+        </image>
+        <a10:id>id</a10:id>
+        <a10:link href="""" />
+        <a10:link xml:base=""http://syndicationlink_url.com/"" syndicationlink_name1="""" d3p1:syndicationlink_name2="""" d3p1:syndicationlink_name3=""syndicationlink_value"" d2p2:syndicationlink_name4="""" rel=""syndicationlink_relationshipType"" type=""syndicationlink_mediaType"" title=""syndicationlink_title"" length=""10"" href=""http://syndicationlink_uri.com/"" xmlns:d3p1=""syndicationlink_namespace"">
+            <Rss20FeedFormatterTests.ExtensionObject xmlns:i=""http://www.w3.org/2001/XMLSchema-instance"" xmlns=""http://schemas.datacontract.org/2004/07/System.ServiceModel.Syndication.Tests"">
+                <Value>10</Value>
+            </Rss20FeedFormatterTests.ExtensionObject>
+        </a10:link>
+        <a10:link rel=""rel_value"" type=""type_value"" title=""title_value"" length=""100"" href=""href_value"" />
+        <Rss20FeedFormatterTests.ExtensionObject xmlns:i=""http://www.w3.org/2001/XMLSchema-instance"" xmlns=""http://schemas.datacontract.org/2004/07/System.ServiceModel.Syndication.Tests"">
+            <Value>10</Value>
+        </Rss20FeedFormatterTests.ExtensionObject>
+        <item>
+            <guid isPermaLink=""false"">id</guid>
+            <description />
+            <a10:updated>0001-01-01T00:00:00Z</a10:updated>
+        </item>
+        <item xml:base=""http://microsoft/relative"" item_name1="""" d3p1:item_name2="""" d3p1:item_name3=""item_value"" d2p2:item_name4="""" xmlns:d3p1=""item_namespace"">
+            <guid isPermaLink=""false"">id</guid>
+            <a10:author />
+            <a10:author author_name1="""" d4p1:author_name2="""" d4p1:author_name3=""author_value"" d2p2:author_name4="""" xmlns:d4p1=""author_namespace"">
+                <a10:name>author_name</a10:name>
+                <a10:uri>author_uri</a10:uri>
+                <a10:email>author_email</a10:email>
+                <Rss20FeedFormatterTests.Extension
```

Comment:
Commented out?

Reply:
Slightly confused, what do you mean? I don't think anything is commented out in this test file

Reply:
L1641 begins with //

Reply:
@hughbe Is this line intended to be commented out?